minifi-toolkit/minifi-toolkit-configuration/src/main/java/org/apache/nifi/minifi/toolkit/configuration/ConfigMain.java

```diff
@@ -336,6 +329,16 @@ public int transform(String[] args) {
         return SUCCESS;
     }
 
+    protected void validateAndPrintIssues(ConfigSchema configSchema) {
+        if (!configSchema.isValid()) {
+            System.out.println("There are validation errors with the template, still outputting YAML but it will need to be edited.");
+            configSchema.getValidationIssues().forEach(System.out::println);
```

Comment:
I don't think the validation issues are getting cleared properly along the way:

![screen shot 2016-10-28 at 4 08 01 pm](https://cloud.githubusercontent.com/assets/11302527/19820707/8646849c-9d28-11e6-840c-d44a5812a78e.png)